src/Raven.Client/Documents/Session/IAsyncDocumentSession.CountersFor.cs

```diff
@@ -8,8 +8,16 @@ namespace Raven.Client.Documents.Session
 {
     public partial interface IAsyncDocumentSession
     {
+        /// <summary>
+        /// Advanced async session API for counter operations on a specific document
+        /// </summary>
+        /// <param name="documentId">The id of the document to operate on</param>
```

Comment:
Can we just inherit `/// <inheritdoc cref="IDocumentSession.CountersFor(string)"/> ` to avoid having it in two places?

Reply:
done